packages/forms/src/UIForm/fieldsets/Fieldset/Fieldset.component.js

```diff
@@ -5,16 +5,18 @@ import Widget from '../../Widget';
 
 export default function Fieldset(props) {
 	const { schema, ...restProps } = props;
-	const { title, items, options } = schema;
+	const { title, items = [], options } = schema;
 
-	return (
+	return items.length ? (
```

Comment:
This make schema.items optional which is not the case in proptypes.
I would like to know how you can have that use case ?

Reply:
Actually I don't know if we have this case but it can happen